maintainers/maintainer-list.nix

```diff
@@ -1439,6 +1439,12 @@
     githubId = 245394;
     name = "Hannu Hartikainen";
   };
+  danderson = {
```

Comment:
The addition to `maintainer-list.nix` should be in a separate commit titled `maintainers: add danderson`